module/VuFind/sql/migrations/pgsql/11.0/012-add-api-key-table.sql

```diff
@@ -0,0 +1,14 @@
+CREATE TABLE api_key (
+  id SERIAL,
+  user_id int NOT NULL,
+  title varchar(255) NOT NULL,
+  token varchar(255) NOT NULL,
+  revoked boolean NOT NULL DEFAULT '0',
+  created timestamp NOT NULL default CURRENT_TIMESTAMP,
+  last_used timestamp NOT NULL default CURRENT_TIMESTAMP,
+  PRIMARY KEY (id),
```

Comment:
Typo: trailing comma should be removed.

```suggestion
  PRIMARY KEY (id)
```